# For rosPackageTemplate
subscriber_topic: /temperature

# For Image converter
camera_name: /csi_cam_0
image_topic: /image_raw

image_publisher: /image_converter/output_video
weed_data_publisher: /vision_target/weed_data

show_img_window: true
